### PR TITLE
vocabulary task les1 has been done

### PR DESCRIPTION
**Pull request by andrpech**

### CODE REVIEW

**Review by olgaz-skiba — APPROVED** [review]
Igfm

**Review by Denis-Pikus — COMMENTED** [review]
lgfm

**Review by Denis-Pikus — APPROVED** [review]
lgfm

File: homework/english/andrii_pechyborshch/lecture1/vocabulary.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+10. Your server enabled. You can continue watching Netflix.
+11. Execute order 66. Ha-ha-ha-ha...
+12. I'm up to implement features I've been working on last two months. 
+13. To install the program please push "install".
+14. You need to plug in a new plugin. 
+15. Github allows you to restore previous versions of the code.
+16. I'm done with this game. It's time to uninstall it.
+17. It's time to update your smartphone.
+18. Upgrading your PC allows you to run heavier games.
+19. Sometimes we all need to be verified for craziness.
```
Comment (goldenas18) [inline homework/english/andrii_pechyborshch/lecture1/vocabulary.md:22]: lgfm

**Review by goldenas18 — APPROVED** [review]

**Review by Alex99905 — COMMENTED** [review]
Lgfm

**Review by Alex99905 — APPROVED** [review]
Lgfm

**Pull request merged**